WhaleSpotting/Controllers/UserController.cs

```diff
@@ -17,13 +17,22 @@ public UserController(ILogger<UserController> logger, ApplicationDbContext conte
         _context = context;
     }
 
-    [HttpGet("{id}")]
+    [HttpGet("{id}/My Sightings")]
```

Comment:
I'm very wary of spaces in URLs - can we make this like below instead?
```suggestion
    [HttpGet("{id}/My-Sightings")]
```

Bonus, when we merge [Megan's PR](https://github.com/TimLeach635/WhaleSpotting-OpenCohort2023/pull/46/files) this page is going to have your user info too, and feels to me slightly more important than the edit page. Would it maybe make sense to actually use the routes below?

1. `{id}` for user page (i.e. put this back to what it was before)
1. `{id}/edit` for the edit page